data/presets/building/waste_shed.json

```diff
@@ -0,0 +1,27 @@
+{
+    "icon": "maki-recycling",
+    "fields": [
+        "{building}",
+        "lockable",
+        "access"
+    ],
+    "geometry": [
+        "area"
+    ],
+    "locationSet": {
+        "include": ["DE", "EE", "SE"]
+    },
+    "tags": {
+        "building": "shed",
+        "amenity": "recycling",
+    },
+    "addTags": {    
+        "access": "private",        
+        "lockable": "yes"
+    },
+    "terms": [
+        "recycling station",
+        "environment station"
+    ],
+    "name": "Waste Shed"
```

Comment:
Maybe "Private Waste Shed" or similar to avoid people applying it to public ones? For example on public collection points?

Reply:
Was thinking maybe "community" or "neighbourhood" or "residential" or "intermediate recycling centre".